packages/ping-protect/README.md

```diff
@@ -0,0 +1,11 @@
+# ping-protect
```

Comment:
This going to be a part of this PR or we will have a followup story for readme?

Reply:
Yeah, let me write up a rough draft of this README, and then I can hand it off to @forgerock-chris for fine tuning.